sample/machineclass.yaml

```diff
@@ -0,0 +1,40 @@
+# Sample machine-class object
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineClass
+metadata:
+  name: small
+  namespace: foo
+capacity:
```

Comment:
leave out capacity / allocatable since those are currently commented out.

Reply:
Done. Thanks for pointing out, I missed it.